client/src/components/SignUp/index.js

```diff
@@ -0,0 +1,162 @@
+import React,{useState} from "react";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import TextField from "@material-ui/core/TextField";
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+import axios from 'axios';
+import {SERVER_URL} from "../../../config";
+import {Redirect} from 'react-router'
+
+const useStyles = makeStyles(theme => ({
+  "@global": {
+    body: {
+      backgroundColor: theme.palette.common.white
+    }
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main
+  },
+  form: {
+    width: "100%", // Fix IE 11 issue.
+    marginTop: theme.spacing(3)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  },
+  errorMessage : {
+    "text-align" : "center",
+    color : "red",
+  }
+}));
+
+export default function SignUp() {
+  const classes = useStyles();
+
+  const [user,setUser] = useState ({name :"", email:"",password: ""});
+  const [errorMessage, setErrorMessage] = useState('');
+  const [signUpSuccess, setSignUpSucess] = useState(false);
+
+  const handleEmailChange = evt => {
+    setUser({...user,email :evt.target.value});
+  };
+
+  const handlePasswordChange = evt => {
+    setUser({...user, password:evt.target.value});
+  };
+
+  const handleNameChange = evt => {
+    setUser({...user, name : evt.target.value});
+  };
+
+  const postData = async (e) => {
+    console.log(user);
+    e.preventDefault();
+    try {
+    const response = await axios({
+      method : 'post',
+      url : `${SERVER_URL}/signup`,
+      data : {...user},
```

Comment:
you can use user object directly

```suggestion
      data: user,
```